python/graphscope/client/session.py

```diff
@@ -98,15 +211,15 @@ class Session(object):
         >>> sess = gs.session()
         >>> g = sess.g()
         >>> pg = g.project(vertices={'v': []}, edges={'e': ['dist']})
-        >>> r = s.sssp(g, 4)
-        >>> s.close()
+        >>> r = gs.sssp(g, 4)
+        >>> sess.close()
```

Comment:
What is `gs` here?

Reply:
defined`import graphscope as gs` in line-208